boxes/README.md

```diff
@@ -1,4 +1,23 @@
-# Aztec Boxes
+# Aztec-App
+
+Aztec-App is a set of tools to ease development on Aztec. It consists of two main components:
+
+## npx script
+
+NPX is a tool bundled with `npm` and other package managers. It allows you to run a binary from a cache without installing it globally.
+
+To ease the development process, Aztec has developed this binary. To run it, install Node and run:
+
+```bash
+npx aztec-app
+```
+
+This will prompt you with some options to clone `Aztec Boxes` and install the sandbox. As the `npx` script evolves, other commands will be added or removed. You can run it with the `-h` flag to know what other commands and flags you can pass to it.
+
+> [!NOTE]  
+> As a tool that doesn't (yet) have automated testing, it versioning and release process is decoupled from `aztec`, and its deployment is entirely manual by running `yarn npm publish --access public`.
```

Comment:
is this deployment do-able by anyone in the org? if not, can you add who has authority and create an issue to make this do-able by anyone in the org?

Reply:
Not really because it requires NPM perms, not github's. But once we're ready to make it into release-please then yes it will use a repo token